search/views.py

```diff
@@ -64,7 +67,18 @@ def do_search(request, course_id=None):
                 raise InvalidPageSize(_('Invalid page size of {page_size}').format(page_size=size))
 
             if "page_index" in request.POST:
-                from_ = int(request.POST["page_index"]) * size
+                page = int(request.POST["page_index"])
+                from_ = page * size
+
+        #Google Analytics - log search request
```

Comment:
Thanks @mulby - for now our requirement is to emit events to the tracking logs - the comment herein is incorrect. Sorry for any confusion.